Tasks/Gradle/task.json

```diff
@@ -334,6 +358,7 @@
         },
         {
             "name": "findbugsAnalysisEnabled",
+            "aliases": [ "findbugsRunAnalysis" ],
```

Comment:
Occurrences like `"aliases": [ "findbugsRunAnalysis" ]` should have "Bugs" capitalized.

Reply:
enableFindBugsRunAnalysis?

Reply:
The verb I was going for in this case was 'run'.  Which do you prefer?  runFindBugsAnalysis?  enableFindBugsAnalysis?

Same with checkstyle.

Reply:
Ah I see it now - area/verb/feature seems to be the pattern. "Run" is good. I feel runFindBugsAnalysis is more readable but you have the "area" before "verb" in other places. Will be better to keep them consistent.

Reply:
👍